pyocd/subcommands/rtt_cmd.py

```diff
@@ -24,6 +26,72 @@
 from pyocd.utility.cmdline import convert_session_options, int_base_0
 from ctypes import Structure, c_char, c_int32, c_uint32, sizeof
 
+# The KBHit code below is based on https://github.com/simondlevy/kbhit by Simon D. Levy,
+# published under the MIT License.
```

Comment:
KBHit should probably be moved to its own `pyocd.utility.kbhit.py` module, so the whole file is MIT-licensed.